scripts/globals/bluemagic.lua

```diff
@@ -7,6 +7,7 @@ TPMOD_CRITICAL = 1;
 TPMOD_DAMAGE = 2;
 TPMOD_ACC = 3;
 TPMOD_ATTACK = 4;
+TPMOD_DURATION = 5;
```

Comment:
I left the semicolon here for now. I'm gonna make a new PR to remove all semicolons from this file, since it violates the style recommendations for Lua.

Reply:
Rebasing and removing semicolons.